securing_apps/topics/oidc/keycloak-gatekeeper.adoc

```diff
@@ -0,0 +1,494 @@
+[[_keycloak_generic_adapter]]
+=== {generic_adapter_name_full}
+
+{project_name} provides a Go programming language adapter for use with OpenID Connect (OIDC) that supports both access tokens in a browser cookie or bearer tokens.
+
+This documentation details how to build and configure {generic_adapter_name} followed by details of how to use each of its features.
+
+For further information, see the included help file which includes a full list of commands and switches. View the file by entering the following at the command line (modify the location to match where you install {generic_adapter_name}):
+
+[source,bash]
+----
+    $ bin/{generic_adapter_name} help
```

Comment:
It seems like variable substitution does not work well inside source code blocks. I'm getting `$ bin/{generic_adapter_name} help` as output. Maybe worth to try something like it was suggested here https://github.com/asciidoctor/asciidoctor/issues/1741#issuecomment-216256423

Reply:
I am trying `[source,bash subs="attributes"]`.